ratis-client/src/main/java/org/apache/ratis/client/RaftClient.java

```diff
@@ -53,8 +52,8 @@
   /** Get the {@link GroupManagementApi} for the given server. */
   GroupManagementApi getGroupManagementApi(RaftPeerId server);
 
-  /** Get the {@link AsyncApi}. */
-  AsyncApi async();
+  /** Get the {@link AsyncRpcApi}. */
+  AsyncRpcApi async();
```

Comment:
We should keep returning AsyncApi in order to hide AsyncRpcApi.  We will cast it to AsyncRpcApi in our code.